tests/phpunit/Checker/Checks/Plugin_Updater_Check_Tests.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * Tests for the Plugin_Updater_Check class.
+ *
+ * @package plugin-check
+ */
+
+use WordPress\Plugin_Check\Checker\Check_Context;
+use WordPress\Plugin_Check\Checker\Check_Result;
+use WordPress\Plugin_Check\Checker\Checks\Plugin_Updater_Check;
+
+class Plugin_Updater_Check_Tests extends WP_UnitTestCase {
+
+	/**
+	 * @dataProvider data_plugin_updater_check
+	 */
+	public function test_run_with_plugin_updater_errors( $type_flag, $plugin_basename, $expected_file, $code, $error ) {
+		$check_context = new Check_Context( TESTS_PLUGIN_DIR . '/tests/phpunit/testdata/plugins/' . $plugin_basename );
```

Comment:
`TESTS_PLUGIN_DIR . '/tests/phpunit/testdata`, should we add this as a const? It seems to be referenced a lot.

Reply:
Added const in https://github.com/10up/plugin-check/pull/189/commits/ddc2211fd668a0e1b04542bd542436acd2bae7d0